oneview/resource_ethernet_network.go

```diff
@@ -35,7 +35,7 @@ func resourceEthernetNetwork() *schema.Resource {
 			"vlan_id": {
 				Type:     schema.TypeInt,
 				Required: true,
-				ForceNew: true,
+				//				ForceNew: true,
```

Comment:
any reason why we have left this line commented?

Reply:
Updated as per comment .